src/utils/file-access-api.ts

```diff
@@ -0,0 +1,142 @@
+/// <reference lib="dom" />
+/**
+ * Provides types for the experimental File System Access API available in Chromium-based browsers
```

Comment:
Nice! In the case of the asset manager, I have different behavior if I have a File vs a FileSystemHandle, so an abstraction like this might not work, though it does look like a fantastic and simple solution :smile: .